testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -8,1976 +8,2028 @@ import pointerMock from "../../../helpers/pointerMock.js";
 import "ui/number_box";
 import "ui/validator";
 
-var NUMBERBOX_CLASS = "dx-numberbox",
-    INVALID_CLASS = "dx-invalid",
-    SPIN_CLASS = "dx-numberbox-spin",
-    SPIN_CONTAINER_CLASS = "dx-numberbox-spin-container",
-    SPIN_UP_CLASS = "dx-numberbox-spin-up",
-    SPIN_DOWN_CLASS = "dx-numberbox-spin-down",
-    TEXTEDITOR_CLASS = "dx-texteditor",
-    INPUT_CLASS = "dx-texteditor-input",
-    CONTAINER_CLASS = "dx-texteditor-container",
-    SPIN_TOUCH_FRIENDLY_CLASS = "dx-numberbox-spin-touch-friendly",
-    PLACEHOLDER_CLASS = "dx-placeholder",
-
-    ACTIVE_STATE_CLASS = "dx-state-active";
-
-
-QUnit.module("basics");
-
-QUnit.test("markup init", function(assert) {
-    var element = $("#numberbox").dxNumberBox();
-
-    assert.ok(element.hasClass(NUMBERBOX_CLASS));
-    assert.ok(element.hasClass(TEXTEDITOR_CLASS));
+const NUMBERBOX_CLASS = "dx-numberbox";
+const INVALID_CLASS = "dx-invalid";
+const SPIN_CLASS = "dx-numberbox-spin";
+const SPIN_CONTAINER_CLASS = "dx-numberbox-spin-container";
+const SPIN_UP_CLASS = "dx-numberbox-spin-up";
+const SPIN_DOWN_CLASS = "dx-numberbox-spin-down";
+const TEXTEDITOR_CLASS = "dx-texteditor";
+const INPUT_CLASS = "dx-texteditor-input";
+const CONTAINER_CLASS = "dx-texteditor-container";
+const SPIN_TOUCH_FRIENDLY_CLASS = "dx-numberbox-spin-touch-friendly";
+const PLACEHOLDER_CLASS = "dx-placeholder";
+const ACTIVE_STATE_CLASS = "dx-state-active";
+
+QUnit.module("basics", {}, () => {
+    QUnit.test("markup init", assert => {
```

Comment:
```suggestion
    QUnit.test("markup init", (assert) => {
```
We agreed to use brackets